inst/rmarkdown/templates/josis/resources/template.tex

```diff
@@ -0,0 +1,173 @@
+% For figures and more when knitting from .Rmd:
+% https://stackoverflow.com/questions/41052687/
+$if(highlighting-macros)$
+$highlighting-macros$
+$endif$
+\providecommand{\tightlist}{%
+  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
+\usepackage{longtable,booktabs,array}
+$if(csl-refs)$
+\newlength{\cslhangindent}
+\setlength{\cslhangindent}{1.5em}
+\newlength{\csllabelwidth}
+\setlength{\csllabelwidth}{3em}
+\newenvironment{CSLReferences}[2] % #1 hanging-ident, #2 entry spacing
+ {% don't indent paragraphs
+  \setlength{\parindent}{0pt}
+  % turn on hanging indent if param 1 is 1
+  \ifodd #1 \everypar{\setlength{\hangindent}{\cslhangindent}}\ignorespaces\fi
+  % set entry spacing
+  \ifnum #2 > 0
+  \setlength{\parskip}{#2\baselineskip}
+  \fi
+ }%
+ {}
+\usepackage{calc}
+\newcommand{\CSLBlock}[1]{#1\hfill\break}
+\newcommand{\CSLLeftMargin}[1]{\parbox[t]{\csllabelwidth}{#1}}
+\newcommand{\CSLRightInline}[1]{\parbox[t]{\linewidth - \csllabelwidth}{#1}\break}
+\newcommand{\CSLIndent}[1]{\hspace{\cslhangindent}#1}
+$endif$
+
+% Article details for accepted manuscripts will be added by editorial staff
+% Omit year if article in press
+% Omit number if article under review
+\josisdetails{%
+   number=N, year=YYYY, firstpage=xx, lastpage=yy,
+  doi={10.5311/JOSIS.YYYY.II.NNN},
+   received={December 24, 2015},
+   returned={February 25, 2016},
+   revised={July 13, 2016},
+   accepted={September 5, 2016}, }
+
+\newcommand{\mydoi}[1]{\href{http://dx.doi.org/#1}{doi:\protect\detokenize{#1}}}
+
+%\renewcommand{\UrlLeft}{http:\sslash}
+%\DeclareUrlCommand\myurl{\def\UrlLeft{}\def\UrlRight{}%
+%\urlstyle{tt}}
+
+\urlstyle{rm}
+\makeatletter
+% Inspired by http://anti.teamidiot.de/nei/2009/09/latex_url_slash_spacingkerning/
+% but slightly less kern and shorter underscore
+\let\UrlSpecialsOld\UrlSpecials
+\def\UrlSpecials{\UrlSpecialsOld\do\/{\Url@slash}\do\_{\Url@underscore}}%
+\def\Url@slash{\@ifnextchar/{\kern-.11em\mathchar47\kern-.2em}%
+    {\kern-.0em\mathchar47\kern-.08em\penalty\UrlBigBreakPenalty}}
+\def\Url@underscore{\nfss@text{\leavevmode \kern.06em\vbox{\hrule\@width.3em}}}
+\makeatother
+
+\hypersetup{
+colorlinks=true,
+linkcolor=black,
+citecolor=black,
+urlcolor=black
+}
+
+% Add the running author and running title information
+\runningauthor{\begin{minipage}{.9\textwidth}\centering $author1$, $author2$\end{minipage}}
+\runningtitle{$runningtitle$}
+
+% Document begins
+\begin{document}
+%\setcounter{page}{33}
+
+
+% Insert your own title
+$if(title)$
+\title{$title$}
+$endif$
+
+% Insert your manuscipts authors, affiliations, and addresses
+$if(author1)$
+\author{$author1$}\affil{$affil1$}
+$endif$
+$if(author2)$
+\author{$author2$}\affil{$affil2$}
+$endif$
+
+\maketitle
+
+% Add 5-10 keywords for every submission
+\keywords{$keywords$}
+
+% Add a short abstract of 150-250 words
+\begin{abstract}
+$abstract$
+\end{abstract}
+
+$body$
+
+$if(bibliography)$
+\bibliographystyle{josisacm}
+\bibliography{$bibliography$}
+$endif$
```

Comment:
I believe this syntax requires to use `natbib` as citation processing engine. 
Currently, your format sets `"default"` meaning use Pandoc processing and biliography seems to be added twice. 
See 
![image](https://user-images.githubusercontent.com/6791940/116248155-e0376c80-a76b-11eb-9e75-29212907a49c.png)


See this is because one of the reference is done using Pandoc's syntax (https://pandoc.org/MANUAL.html#citations) and the others are made using `\cite` I believe so Pandoc 
does not process them. 

So maybe you need to default to "natbib" for this format. 

Usually with Pandoc a CSL is used by I don't see one for ACM Josis
* https://bookdown.org/yihui/rmarkdown-cookbook/bibliography.html
* https://www.zotero.org/styles?q=ACM